buildSrc/src/main/java/Config.kt

```diff
@@ -11,7 +11,7 @@ object Config {
         private val sdkVersion = 29
 
         val buildToolsVersion = "29.0.2"
-        val minSdkVersion = 14
+        val minSdkVersion = 21
```

Comment:
we are losing linter here, this should be done on specific ndk build file

Reply:
> Manifest merger failed : uses-sdk:minSdkVersion 14 cannot be smaller than version 21 declared in library [:sentry-android-ndk] /Users/bruno/git/sentry-android/sentry-android-ndk/build/intermediates/library_manifest/debug/AndroidManifest.xml as the library might be using APIs not available in 14
        Suggestion: use a compatible library with a minSdk of at most 14,
                or increase this project's minSdk version to at least 21,
                or use tools:overrideLibrary="io.sentry.android.ndk" to force usage (may lead to runtime failures)